packages/plugin-react-swc/src/index.ts

```diff
@@ -144,6 +150,23 @@ const react = (_options?: Options): PluginOption[] => {
             '[vite:react-swc] The MDX plugin should be placed before this plugin',
           )
         }
+
+        /*
+         * Suggest to use vite-plugin-react-oxc if `rolldown-vite` is used and:
+         * No swc plugins are set
+         * mutateSwcOptions is not set
+         * It is not disabled by the user
+         */
```

Comment:
This is is pretty clear to me, I don't think we need to maintain twice the list in the future

Reply:
Yes personally I would prefer to remove it

Reply:
Removed both comments 👍